docs/syntax_and_semantics/platform_support.md

```diff
@@ -19,6 +19,7 @@ and drop into *Tier 2*.
 
 | Target | Description | Supported versions | Comment |
 | ------ | ----------- | ------------------ | ------- |
+| `aarch64-darwin` | Aarch64 macOS<br> (Apple Silicon) | 11+ | :material-selection-marked-circle: tests<br> :material-checkbox-marked-circle: builds
```

Comment:
Do we need to call out we're only testing 14 atm given that's the only hosted runner that uses ARM series?

Similarly below for x86_64, that is now only 13, not 11.